resources/dicts/events/death/general.json

```diff
@@ -4514,7 +4722,7 @@
         ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c lost a life saving r_c from a snake.",
+        "event_text": "m_c saved r_c from a snake and was bitten in the process.",
```

Comment:
let's specify that they died from the bite, otherwise players might think they were meant to just get the snake bite injury